tokio-threadpool/src/lib.rs

```diff
@@ -1300,6 +1345,9 @@ impl Worker {
                 // Enter an execution context
                 let mut enter = tokio_executor::enter().unwrap();
 
+                #[cfg(feature = "futures-0-2")]
+                let _enter2 = futures2::executor::enter();
```

Comment:
Should `tokio_executor::enter()` include `futures2::executor::enter()` under the hood instead of entering both?